src/hooks/useClickOutsideHandler.js

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useEffect } from "react";
+
+const useClickOutsideHandler = (ref, callback) => {
+  const handleClickOutside = useCallback(
+    (e) => {
+      if (ref.current && !ref.current.contains(e.target)) {
+        callback();
+      }
+    },
+    [ref, callback]
+  );
+
+  useEffect(() => {
+    document.addEventListener("click", handleClickOutside);
```

Comment:
We found that `document.addEventListener` does not work fine with IFrames, and we need IFrames in some places. For that reason, we are using [react-cool-onclickoutside](https://github.com/wellyshen/react-cool-onclickoutside) in another project. See here the related PR: https://github.com/FromDoppler/doppler-editors-webapp/pull/136

Reply:
Library `react-cool-onclickoutside` implemented.
Custom hook `useClickOutsideHandler` deleted.
Thanks!